lib/http_server_fixture/routes.ex

```diff
@@ -45,16 +47,8 @@ defmodule HTTPServerFixture.Routes do
       "/xml_response" => %{
         handler: HTTPServerFixture.XMLResponse,
         methods: ["GET"]
-      },
-      "/health-check.html" => %{
-        handler: HTTPServer.Router.Handlers.ServeStatic,
-        methods: ["GET"],
-        filepath: "test/http_server_spec/web/health-check.html"
-      },
-      "/layout-styles.css" => %{
-        handler: HTTPServer.Router.Handlers.ServeStatic,
-        methods: ["GET"],
-        filepath: "test/http_server_spec/web/layout-styles.css"
       }
     }
+    |> ServeStatic.add_static_dir("test/http_server_spec/web")
```

Comment:
what happens if you provide a "garbage path"? Perhaps some unit tests for happy-path and error scenarios should be included in the PR?

Reply:
Just pushed an update that creates a unit test for this! If the path is bad, `Path.wildcard/1` returns an empty map which means attempting to access those static files returns a 404.